### PR TITLE
Implement /lister/{id}/logg API endpoint

### LINKED ISSUE #15: Return recent checkins for /lister/{id}/logg

### PR DESCRIPTION
**Pull request by Starefossen**
* [x] Checkin.getCheckinsForList() static method
* [x] Implement `/lister/{id}/logg` API endpoint
* [x] Documentation
  * Document in README.md
  * Document in `/`
* [x] Add Nasjonal Turbase config to Docker Cloud
     * `NTB_API_ENV`
     * `NTB_API_KEY`

Close #15

### CODE REVIEW

**Comment by Starefossen** [comment]
Rebase på origin/master.

File: index.js [hunk 1]
```diff
@@ -151,8 +154,33 @@ router.get('/steder/:sted/besok/:checkin', (req, res, next) => {
 });
 
 router.get('/lister/:liste/stats', notImplementedYet);
-router.get('/lister/:liste/logg', notImplementedYet);
-router.post('/lister/:liste/blimed', notImplementedYet);
+
+router.get('/lister/:liste/logg', (req, res) => {
+  Checkin.getCheckinsForList(req.params.liste)
+    .then(checkins => {
+      res.json({ checkins });
```
Comment (Starefossen) [inline index.js]: Should we stick to the pattern of wrapping body in `data` property?

```
res.json({ data: checkins });
```

Comment (geekus) [inline index.js]: Sounds like a good idea.

**Comment by Starefossen** [comment]
Have updated config in Docker Cloud

**Pull request merged**